### DISCUSSION TITLE
Everytime my lib file is missing

### DISCUSSION THREAD

**Discussion by sanu2002** [Cyfrin/foundry-full-course-cu#2476] [Q&A]
Wen ever i am running the forge init hello_foundry ,it showing the error like this :-

I am using gitbash terminal ..

```
$ forge init hello_foundry
Error: 
The target directory is a part of or on its own an already initialized git repository,
and it requires clean working and staging areas, including no untracked files.

Check the current git repository's status with `git status`.
Then, you can track files with `git add ...` and then commit them with `git commit`,
ignore them in the `.gitignore` file, or run this command again with the `--no-commit` flag.

If none of the previous steps worked, please open an issue at:
https://github.com/foundry-rs/foundry/issues/new/choose


```
```
![forge](https://github.com/user-attachments/assets/5f8ba0b4-3506-42c8-a70e-ce6e4e9348b6)
```

If anyone know this issue please feel free to help me !!!

**Comment by EngrPips** [comment]
Hello @sanu2002, Have you tried all the suggested solutions by Foundry?

**Reply by sanu2002** [reply]
> Hello @sanu2002, Have you tried all the suggested solutions by Foundry?

yes, but still it is not showing the expected resullt .

**Reply by sanu2002** [reply]
> Hello @sanu2002, Have you tried all the suggested solutions by Foundry?
i guess there is some issue in gitbash terminal

**Reply by EngrPips** [reply]
I think this `If none of the previous steps worked, please open an issue at:
https://github.com/foundry-rs/foundry/issues/new/choose` is the next line of action. Why can't you create a new `hello_foundry` directory manually and just run `forge init` inside of it.

**Reply by sanu2002** [reply]
> I think this `If none of the previous steps worked, please open an issue at: https://github.com/foundry-rs/foundry/issues/new/choose` is the next line of action. Why can't you create a new `hello_foundry` directory manually and just run `forge init` inside of it.

Okay, I’ll keep that in mind next time. However, I have tried all possible solutions, but none of them are working .

**Reply by EngrPips** [reply]
Let's look forward to someone that knows a solution that will work.

**Comment by sanu2002** [comment]
After installing the Remote Development extension from the Extensions tab, a pop-up appeared in the bottom-left corner prompting me to connect with WSL. After connecting, a Bash terminal opened, and everything is working as expected.

This is the solution for window user !!!

**Reply by EngrPips** [reply]
Awesome, Thanks for the heads-up, you should mark this discussion as answered by you.
